pages/common/bosh.md

```diff
@@ -0,0 +1,35 @@
+# bosh
+
+> Cli tool to deploy and manage the bosh director.
+
+- Deploy the director:
+
+`bosh create-env {{bosh.yml}} --state={{state.json}} --vars-store={{creds.yml}} -v {{custparam}}`
+
+- Create local alias for director:
+
+`bosh alias-env {{name}} -e {{192.168.1.123}} --ca-cert {{$(bosh int ./creds.yml --path /director_ssl/ca)}}`
+
+- Login to the director:
+
+`bosh login -e {{environment}} --client={{username}} --client-secret={{password}}`
```

Comment:
Is there no option for the command to prompt for a password ? Writing passwords in the command line is generally considered bad practice.

Reply:
agreed, if a secret and password are not provided it will prompt the user for them. i will change the MD to reflect this.